python_modules/libraries/dagster-dbt/tox.ini

```diff
@@ -12,6 +12,7 @@ deps =
   -e ../dagster-duckdb-pandas
   dbt_15X: dbt-core==1.5.*
   dbt_15X: dbt-duckdb==1.5.*
+  dbt_15X: duckdb<0.10.0
```

Comment:
@rexledesma  are you ok if i put this pin on the duckdb version for the dbt 15 tests? I couldn't find a reason why, but dbt 1.5 doesn't seem to be compatible with duckdb 0.10.0